services/awsgateway/lambda.go

```diff
@@ -0,0 +1,63 @@
+package awsgateway
+
+import (
+	"encoding/json"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/lambda"
+	"github.com/iotaledger/giota"
+)
+
+const (
+	// Public
+	// https://docs.aws.amazon.com/lambda/latest/dg/limits.html
+	// 6MB payload, 300 sec execution time, 1000 concurrent exectutions.
+	// Limit to 1000 POSTs and 20 chunks per request.
+	MaxConcurrency = 1000
+	MaxChunksLen   = 3000 // 3 MB
```

Comment:
@rfornea No matter what I change this to, the amount of chunks returned from the query in `process_unassigned_chunks` seems to max out to 300 chunks. Any ideas why?

Reply:
It's because of the batch size.  I believe this is configured in the job_runner file.